src/Concerns/AddsIncludesToQuery.php

```diff
@@ -78,6 +78,16 @@ protected function ensureAllIncludesExist()
             throw InvalidIncludeQuery::includesNotAllowed($diff, $allowedIncludeNames);
         }
 
-        // TODO: Check for non-existing relationships?
+        $methods = get_class_methods($this->getModel());
+
+        $methods = collect($methods)->reject(function ($method) {
```

Comment:
`$methods` can be inlined in `collect()` here to avoid confusion over declaring the same variable twice :)